### DISCUSSION TITLE
Is it actually possible to win the Price of Loyalty mission 2 in normal difficulty?

### DISCUSSION THREAD

**Discussion by HubKing** [ihhub/fheroes2#6806] [Q&A] [answered]
It feels almost impossible. I have watched some YouTube videos but they were using the original version, and it appeared to me that the number of enemy heroes is a lot smaller and their troops are also weaker. In fHeroes, if I reach the top-left area in week 2, there are about 4 enemy heroes and at least  2 of them have large troops. Even if I beat one of them, another one comes and defeats my hero whose troops has been reduced by the previous battle. Isn't the A.I. too strong? Could anyone win this mission in normal difficulty?

**Comment by Alucard648** [comment, marked as answer]
Same thing + enemy heroes have better stats. On top of that I got attacked on week 3 and nuked with DragoGeddon.

**Reply by zenseii** [reply]
I've won this on Hard difficulty on version 1.0.1 public so Normal shouldn't be too hard. 

Here's someone beating it on hard on 1.0.0, which isn't very different from 1.0.1 from an AI perspective.

https://www.youtube.com/watch?v=ZwpAzUwM_L8&ab_channel=TheShadowRooster
Either you do like the youtuber and steal a boat, or you use the more predictable tactic of focusing on getting unicorns by week 2 and cold ray/lightning bolt on your sorceress and the relevant artifacts. Get the gold mine day 1. Grab Red's north western town by week 2/early week 3. Don't engage in any battles that you lose too many troops on. Don't attack the yellow main hero that has lvl5 spells unless you can avoid him retreating and not lose many troops.

The second part of this map is about making sure you have enough troops on your main hero to work as a deterrent so that they won't attack you while you grab their castles.

**Reply by Alucard648** [reply]
> Don't attack the yellow main hero that has lvl5 spells unless you can avoid him retreating and not lose many troops.

Yes, that guy attacked me on week 3 and nuked my army with Armageddon spell, while having green dragons only.

**Reply by HubKing** [reply]
That YouTube guy sucked and he only got lucky because a weak enemy happened to came in a boat. In my cases, they come with black dragon(s).
Acquiring one enemy castle at week 2 is not hard, but how to move from there or get reinforcement? Because red/yellow heroes are constantly roaming around the region, so the main hero cannot go out, and the secondary hero cannot pass the narrow path without getting attacked to reach the main hero.

**Reply by zenseii** [reply]
> That YouTube guy sucked 

All the more proof that this map is beatable by skilled players.

> he only got lucky because a weak enemy happened to came in a boat.

Sometimes you have luck and other times you meet more obstacles in this game. Obviously it's not a valid strategy to rely on luck.

> Acquiring one enemy castle at week 2 is not hard, but how to move from there or get reinforcement? 

At this point I usually trade castles with Red and get their main barbarian castle to the north. After this it's mainly about choosing what heroes to take out so that they lose 2500 gold and then eventually kill off Red. If you see yellow come with a large army you should be able to walk past it and offer them your castles as a trade for their warlock castles. Then you can block the portal as usual with a hero, and hope that they don't show up with a boat. You could also block them on the island if you manage to lure them there.

As for the secondary hero I only move them up when I have phoenixes, in other words getting the mercury and alchemist lab near the narrow path from your castle will be essential. Try to get the ore mine to the east of the northernmost barbarian castle.

**Reply by HubKing** [reply]
Finally, I have figured it out. The only way that worked is as the following:

Up until week 3, take the mercury and other resources on the left, but don't attack any castle. Then, come back to the starting castle and upgrade the magi guilt to 2 to get "Summon Boat". Don't waste money on upgrading castle or building Red Tower, etc, because this castle is not defensible any way, and we later need money.  Around the 4th week, summoning boat begins to succeed. I don't know the exact reason, but maybe because enemy has built a shipyard.  Go straight to the right island where two Warlock castles are. The castles are not guarded (or very few guards), and they have a ton of centaurs, and other troops. Buy them with saved money. Buy a disposable hero and send him through the dimension gate. It lead to the upper right warlock castle. Get the vision near the castle even if he gets killed. Main hero waits in the island until the yellow player leaves the castle. When the castle is empty, send the main hero there and take the castle. If the timing is right, the yellow player has built a green or red tower, but he has not bought dragons yet, so there are several dragons to buy. 

Keep guarding the three Warlock castles from other players, and accumulate non-dragon troops. When the army becomes strong enough,  take the  sulphur mine near the middle knight castle. Sending the hero from the Warlock castle to the knight castle on land takes too much time, and it can be reach in one day through the sea, if he has "Navigation". Develop a second hero and place him on the Knight castle near the sulphur mine can help to always retake the mine.

Also use market to trade almost anything for sulphur. With enough sulphur, we can eventually buy black dragons, and by then the game becomes easy. We can take the remaining castle easily with the dragons and win.

**Reply by oleg-derevenetz** [reply]
> Around the 4th week, summoning boat begins to succeed. I don't know the exact reason, but maybe because enemy has built a shipyard.

BTW that trick with stealing the enemy boats will not work anymore in 1.0.2. Only neutral boats can be stolen.

**Reply by HubKing** [reply]
Is summoning stealing enemy boats, not getting an abandoned one? Because boats do not seem to have any indication of owners (like flags). Anyway, if summoning boats does not work, we cannot use boats because there is only one castle that can have a shipyard, and it is the yellow's right-middle Warlock castle. Then there are two alternative ways:

1. Go to the dimension gate near that castle on land, but it will take a lot of time and we can get attacked by an enemy. Warp to the island through the gate.
2. Hope a weak hero of the Yellow come to our starting castle by boat at around week 5~6, so that we could defeat him and get his boat to go to the island.

**Reply by oleg-derevenetz** [reply]
> Is summoning stealing enemy boats, not getting an abandoned one? Because boats do not seem to have any indication of owners (like flags).

Although they do not have flags, they still belong to a particular player (see #4109 for details). Neutral are only the boats that the author of the map placed on it initially and only until some hero boarded it.

**Reply by HubKing** [reply]
Come to think of it, it makes sense that player 1 cannot summon the boat that player 2, his enemy, has just built in a remote castle. Anyway, the Flatpak package was just updated to 1.0.2 and I tested if I could win without summoning boat. Short answer: yes.

1. First thing I tried: Just staying in the starting castle accumulating troops until enemy comes by a boat. Some 3~4 months passed and the enemy who came by boat had 20 black dragons --> fail
2. Next strategy I tried: With week 3's army, I went straight to the far-right warlock castle. It was nearly empty, so I took it, bought army there, and used the warp gate to go to the island. A relatively weak yellow hero was there, so I could defeat him. One castle had green tower and the other castle had level-5 magi guild with dimensional door. Both had a lot of troops to purchase, so after some time, I took the sulphur mine by moving my main hero using dimensional door from the far-right Warlock castle to the mine. After accumulating sulphur, with about 20 black dragons, I could take all other castles and win.

**Comment by ihhub** [comment]
A short answer: **yes**.
